src/artemis/move_det_robot_load.py

```diff
@@ -0,0 +1,44 @@
+import bluesky.plan_stubs as bps
```

Comment:
Should: I think this is part of https://github.com/DiamondLightSource/python-artemis/issues/335 right? Can we do it in a separate PR for that rather than here?

Reply:
yes, not sure how that sneaked in there

Reply:
You can remove it by doing `git rm --cached src/artemis/move_det_robot_load.py`. This will take it out of git but keep it locally if you're worried about losing it.